.env.example

```
@@ -17,31 +17,31 @@ ARBITRUM_OUTBOX=
 ARBITRUM_ROOTS_SLOT=
 
 #Linea Envs
-LINEA_CHAIN_ID=
-LINEA_ROLLUP=
-LINEA_STATE_ROOT_HASHES_SLOT=
-LINEA_L2_MESSAGE_SERVICE=
+LINEA_CHAIN_ID=59141
+LINEA_ROLLUP=0xB218f8A4Bc926cF1cA7b3423c154a0D627Bdb7E5
+LINEA_STATE_ROOT_HASHES_SLOT=282
+LINEA_L2_MESSAGE_SERVICE=0x971e727e956690b9957be6d51Ec16E73AcAC83A7
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The LINEA_L2_MESSAGE_SERVICE key should go before the LINEA_ROLLUP key

(UnorderedKey)